core-bundle/tests/Search/DocumentTest.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Tests\Search;
+
+use Contao\CoreBundle\Search\Document;
+use Nyholm\Psr7\Uri;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class DocumentTest extends TestCase
+{
+    /**
+     * @dataProvider documentProvider
+     */
+    public function testDocument(string $body, array $expectedJsonLds): void
```

Comment:
This method name does not describe what the method does.

Reply:
Yeah, it just tests everything. No idea. Feel free to rename if you have a better idea.